Tests/MapboxMapsTests/Camera/BasicCameraAnimatorTests2.swift

```diff
@@ -0,0 +1,166 @@
+import XCTest
+@testable import MapboxMaps
+
+final class BasicCameraAnimatorTests: XCTestCase {
+
+    var impl: MockBasicCameraAnimator!
+    var animator: BasicCameraAnimator!
+    var delegate: MockCameraAnimatorDelegate!
+
+    override func setUp() {
+        super.setUp()
+        impl = MockBasicCameraAnimator()
+        animator = BasicCameraAnimator(impl: impl)
+        delegate = MockCameraAnimatorDelegate()
+        animator.delegate = delegate
+    }
+
+    override func tearDown() {
+        delegate = nil
+        animator = nil
+        impl = nil
+        super.tearDown()
+    }
+
+    func testOwner() {
+        impl.owner = .random()
```

Comment:
Should the opposite to be tested as well? 

The current test tests the setter of the property on impl side and the getter on animator side.

While the opposite would cover the setter of the property on the animator side and the getter on impl side.

```swift 
animator.owner = .random()
XCTAsseartEqual(animation.owner, impl.owner)
```

Reply:
This test is one-way since `animator.owner` is read-only.